__docs__/wonder-blocks-theming/tokens-font.mdx

```diff
@@ -0,0 +1,154 @@
+
+export const sampleTextUpper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+export const sampleTextLower = "abcdefghijklmnopqrstuvwxyz";
+
+export const baseColumns = (kind) => [
+    {
+        label: "Token",
+        cell: (row) => (
+            <code>
+                font.{kind}.{row.label}
+            </code>
+        ),
+    },
+    {
+        label: "Value",
+        cell: "value",
+    },
+];
+
+export const mapFunction = ([key, value]) => ({
+    label: key,
+    value: value,
+});
+
+export const tokensFamilyMap = Object.entries(tokens.font.family).map(
+    mapFunction,
+);
+export const tokensWeightMap = Object.entries(tokens.font.weight).map(
+    mapFunction,
+);
+export const tokensSizeMap = Object.entries(tokens.font.size).map(mapFunction);
+export const tokensLineHeightMap = Object.entries(tokens.font.lineHeight).map(
+    mapFunction,
+);
+
+# Tokens
+
+## Typography
+
+Text sizes, weights and font families used across Wonder Blocks.
+
+<Banner
+    kind="warning"
+    text="Note that this is intended only internal
+    development purposes. We'll let you know as soon as this is ready for public
+    consumption."
+/>
+
+### Family
+
+The available font families in our system.
+
+<TokenTable
+    columns={[
+        ...baseColumns("family"),
+        {
+            label: "Example",
+            cell: (row) => (
+                <View
+                    style={{
+                        fontFamily: row.value,
+                    }}
+                >
+                    <span>{sampleTextUpper}</span>
+                    <span>{sampleTextLower}</span>
+                </View>
+            ),
+        },
+    ]}
+    data={tokensFamilyMap}
+/>
+
+### Size
+
+The list of possible font sizes that can be used with our components.
+
+<TokenTable
+    columns={[
+        ...baseColumns("size"),
+        {
+            label: "Example",
+            cell: (row) => (
+                <View
+                    style={{
+                        fontSize: row.value,
+                        lineHeight: 1,
+                    }}
+                >
+                    <span>{sampleTextUpper}</span>
+                    <span>{sampleTextLower}</span>
+                </View>
+            ),
+        },
+    ]}
+    data={tokensSizeMap}
+/>
+
+### LineHeight
+
+The vertical space associated to a given font size.
+
+<TokenTable
+    columns={[
+        ...baseColumns("lineHeight"),
+        {
+            label: "Example",
+            cell: (row) => (
+                <View
+                    style={{
+                        backgroundColor: tokens.color.offBlack8,
+                        lineHeight: `${row.value}px`,
+                    }}
+                >
+                    <span>{sampleTextUpper}</span>
+                    <span>{sampleTextLower}</span>
+                </View>
+            ),
+        },
+    ]}
+    data={tokensSizeMap}
+/>
+
+### Weight
+
+Font weight options include regular, bold and black.
```

Comment:
it looks like the actual options are `light`, `regular` and `bold` now? this description seems outdated.